packages/vhd-lib/Vhd/VhdAbstract.js

```diff
@@ -282,7 +282,7 @@ exports.VhdAbstract = class VhdAbstract {
     const rawHeader = fuHeader.pack(header)
     checksumStruct(rawHeader, fuHeader)
 
-    assert.strictEqual(offset % SECTOR_SIZE, 0)
+    assert.strictEqual(offset % SECTOR_SIZE, 0, `offset should be aligned to SECTORE SIZE : ${offset} `)
```

Comment:
Normally we already fixed this.

```suggestion
    assert.strictEqual(offset % SECTOR_SIZE, 0, `offset should be aligned to SECTOR_SIZE: ${offset}`)
```